Non existent files causing build failure

Downloaded hrp-master.zip today (2019-10-04), can't find a git repo id or other identifier in the unzipped file to narrow the version down further.

[  7%] Building CXX object hrp-master/am_driver_legacy/CMakeFiles/am_driver_legacy_node.dir/src/automower_legacy.cpp.o
In file included from /home/ben/catkin_ws_2/src/hrp-master/hq_decision_making/src/hq_ROSTask.cpp:20:0:
/home/ben/catkin_ws_2/src/hrp-master/hq_robot_task/include/hq_robot_task/hq_RobotTask.h:47:46: fatal error: hq_robot_task/hq_RobotTaskAction.h: No such file or directory
compilation terminated.

--- a/am_driver_legacy/CMakeLists.txt
+++ b/am_driver_legacy/CMakeLists.txt
@@ -110,7 +110,7 @@ add_executable(am_driver_legacy_node src/am_driver_legacy_node.cpp src/automower
 
 ## Add cmake target dependencies of the executable/library
 ## as an example, message headers may need to be generated before nodes
-# add_dependencies(am_driver_node am_driver_generate_messages_cpp)
+add_dependencies(am_driver_legacy_node am_driver_generate_messages_cpp)
 
 ## Specify libraries to link a library or executable target against
 target_link_libraries(am_driver_legacy_node

--- a/am_driver_safe/CMakeLists.txt
+++ b/am_driver_safe/CMakeLists.txt
@@ -120,7 +120,7 @@ add_executable(am_driver_safe_node
 
 hq_decision_making_parsing(src/automower_safe_states.cpp)
 
-add_dependencies(am_driver_safe_node am_driver_safe_generate_messages_cpp)
+add_dependencies(am_driver_safe_node am_driver_generate_messages_cpp am_driver_safe_generate_messages_cpp)
 
 ## Add cmake target dependencies of the executable/library
 ## as an example, message headers may need to be generated before nodes

--- a/hq_decision_making/CMakeLists.txt
+++ b/hq_decision_making/CMakeLists.txt
@@ -114,6 +114,7 @@ include_directories(
 #===================== ROS =============================================
 
 add_library(hq_decision_making_ros src/hq_ROSTask.cpp)
+add_dependencies(hq_decision_making_ros hq_robot_task_generate_messages_cpp)
 target_link_libraries(hq_decision_making_ros ${catkin_LIBRARIES} )
 
 #=======================================================================

